.github/workflows/visualizations.yml

```diff
@@ -0,0 +1,99 @@
+name: "Visualizations"
+
+on:
+  push:
+    paths-ignore:
+      - "packages/api-client/**"
+
+jobs:
+  test:
+    name: Build and test on Node ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ["14.x", "16.17.1"]
+        os: [ubuntu-latest]
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Node ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      
+      - name: Install
+        run: npm ci
+
+      - name: Bootstrap
+        run: npm run bootstrap
+
+      - name: Test
+        run: npm test
+
+  lint:
+    name: Lint on Node ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ["16.17.1"]
+        os: [ubuntu-latest]
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Node ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Install
+        run: npm ci
+
+      - name: Bootstrap
+        run: npm run bootstrap
+
+      - name: Lint
+        run: npm run lint
+
+  chromatic-deployment:
+    if: github.event.pull_request.draft == false
```

Comment:
Still run even if the PR in draft mode, see https://github.com/opendatasoft/ods-dataviz-sdk/pull/214